node/task.ts

```diff
@@ -457,7 +457,7 @@ export function getDelimitedInput(name: string, delim: string, required?: boolea
 export function filePathSupplied(name: string): boolean {
     // normalize paths
     var pathValue = this.resolve(this.getPathInput(name) || '');
-    var repoRoot = this.resolve(this.getVariable('build.sourcesDirectory') || '');
+    var repoRoot = this.resolve(this.getVariable('build.sourcesDirectory') || this.getVariable('System.DefaultWorkingDirectory') || '');
```

Comment:
I thought there is a variable that means differently between Build and RM, should we just use that one?